pytest_inmanta/plugin.py

```diff
@@ -659,6 +660,18 @@ def __init__(
         self._handlers: typing.Set[ResourceHandler] = set()
         config.Config.load_config()
 
+    def _synchronize_interpreters(self) -> None:
```

Comment:
I don't really like this name. `_set_sys_executable` is a lot more clear in my opinion.